src/ui/components/Navigation/Core/FavoritesNameWithAliasItem/FavoritesNameWithAliasItem.tsx

```diff
@@ -0,0 +1,56 @@
+import React from 'react';
+
+import {Lock, Tag} from '@gravity-ui/icons';
+import {Icon, Popover} from '@gravity-ui/uikit';
+import block from 'bem-cn-lite';
+import {ENTRY_CONTEXT_MENU_ACTION} from 'ui/components/EntryContextMenu';
+
+import {MenuClickArgs} from '../../types';
+
+import './FavoritesNameWithAliasItem.scss';
+
+const b = block('dl-core-navigation-favorites-alias');
+
+type FavoritesNameWithAliasItemProps = {
+    entryId: string;
+    name: string;
+    alias: string;
+    isLocked: boolean;
+    onMenuClick: (args: MenuClickArgs) => void;
+};
+
+export const FavoritesNameWithAliasItem = (props: FavoritesNameWithAliasItemProps) => {
+    const {name, alias, isLocked, onMenuClick} = props;
+
+    const onLabelClick = (e: React.MouseEvent<HTMLDivElement, MouseEvent>) => {
+        e.preventDefault();
+        e.stopPropagation();
+
+        onMenuClick({
+            entry: {entryId: props.entryId, alias: props.alias},
+            action: ENTRY_CONTEXT_MENU_ACTION.EDIT_FAVORITES_ALIAS,
+        });
+    };
+
+    const text = alias === '' ? name : alias;
+    const isAliasVisible = alias !== '';
+
+    return (
+        <>
+            <div title={name} className={b('name-line')}>
```

Comment:
Let's use `title={text}` for showing correct title:
<img width="829" alt="image" src="https://github.com/datalens-tech/datalens-ui/assets/29424746/81b821eb-70dc-4aa1-a5e4-91cf53b187cb">

Reply:
Discussed offline, fixed